packages/gatsby-link/src/is-local-link.js

```diff
@@ -0,0 +1,11 @@
+// Copied from https://github.com/sindresorhus/is-absolute-url/blob/3ab19cc2e599a03ea691bcb8a4c09fa3ebb5da4f/index.js
+const ABSOLUTE_URL_REGEX = /^[a-zA-Z][a-zA-Z\d+\-.]*?:/
+const isAbsolute = path => ABSOLUTE_URL_REGEX.test(path)
+
+export const isLocalLink = path => {
+  if (typeof path !== `string`) {
+    throw new TypeError(`Expected a \`string\`, got \`${typeof path}\``)
+  }
```

Comment:
This is behaviour change - it seems like we did allow `undefined` before and now it throws error? I think it make sense in general to require path, but I'd say this will need for v5

Reply:
Fixed